docs/user-guide/configmap/index.md

```diff
@@ -362,7 +362,7 @@ spec:
   containers:
     - name: test-container
       image: gcr.io/google_containers/busybox
-      command: [ "/bin/sh", "cat", "/etc/config/special.how" ]
```

Comment:
instead how about just adding "-c" as shown in the other commands that use /bin/sh.  Also would you update the next occurrence as well.